lib/_stream_readable.js

```diff
@@ -1209,40 +1210,8 @@ function endReadableNT(state, stream) {
 }
 
 Readable.from = function(iterable, opts) {
```

Comment:
could we just do `Readable.from = require('internal/streams/from')` now that we have core snapshots?

Reply:
I need to backport this to 10, I don't think we have those there yet.
We can remove this type of "postponed loading" everywhere I think - would you mind to open an issue? We'd need to verify it does not slow thing down.